uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml

```diff
@@ -73,6 +73,8 @@ Page {
      */
     property var popupManager: null
 
+    property var popup: null
```

Comment:
I added it to this issue to create an issue for when all of these are completed to go and update that doc since I was thinking we wouldn't want to update the doc yet until the PopupView + PopupElements is completed.